olca-app/src/org/openlca/app/tools/graphics/model/Side.java

```diff
@@ -1,6 +1,6 @@
 package org.openlca.app.tools.graphics.model;
 
 public record Side() {
```

Comment:
better make this an enum, there is not really a performance difference and it is more type safe